test/RunTests

```diff
@@ -2323,4 +2323,13 @@ train-sets/ref/slates_simple_w_interactions.stderr
 {VW} -d train-sets/squarecb_load.dat --cb_explore_adf -q UA --squarecb -i models/sqcb_ld.model -t
     train-sets/ref/squarecb_load.stderr
 
+# Test 316: bfgs preconditioner training
```

Comment:
[nit] I do not think we can say "preconditioner training". Probably something like "bfgs preconditioner overflowing test" is more accurate description